src/main/java/com/eternalcode/combat/region/RegionService.java

```diff
@@ -0,0 +1,39 @@
+package com.eternalcode.combat.region;
+
+import com.eternalcode.combat.bridge.WorldGuardBridge;
+import com.sk89q.worldedit.bukkit.BukkitAdapter;
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.protection.ApplicableRegionSet;
+import com.sk89q.worldguard.protection.regions.RegionContainer;
+import com.sk89q.worldguard.protection.regions.RegionQuery;
+import org.bukkit.Location;
+
+import java.util.List;
+
+public class RegionService {
+
+    private final WorldGuardBridge worldGuardBridge;
```

Comment:
Przydało by się tutaj zastosować odwrócenie zależności, aby domena nie dotykała WG i WE